addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -695,6 +699,10 @@ var SnippetEditor = Widget.extend({
             }
         }
 
+        if (this.excludeDropInSelector.length) {
+            $selectorChildren = $selectorChildren.not(this.excludeDropInSelector.join(", "));
```

Comment:
Kinda bad to have to do a join each time while the array is not going to ever change. The joined selectors should really be the variable saved in `this`. Although, instead of this, should not the functions linked to  `selectorChildren` be impacted directly instead of this? So that the call to `selectorChildren[i].all()` just above this directly include this exclusion?